docs/generators/fsharp-functions.md

```diff
@@ -34,7 +34,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |prependFormOrBodyParameters|Add form or body parameters to the beginning of the parameter list.| |false|
 |sortModelPropertiesByRequiredFlag|Sort model properties to place required parameters before optional parameters.| |true|
 |sortParamsByRequiredFlag|Sort method arguments to place required parameters before optional parameters.| |true|
-|sourceFolder|source folder for generated code| |OpenAPI/src|
+|sourceFolder|source folder for generated code| |OpenAPI\src|
```

Comment:
Thanks for the PR. Are you using Git Bash when updating the samples?

I've never been able to repeat the issue in my Windows laptop so I would appreciate if you can share more about your development environment/setting.

Reply:
Indeed, I am on Windows 11 and using Git Bash to update the samples. I just followed the instructions in the pull request template, but according to my understanding, none of the samples should actually be affected by my edits. I would be happy to try regenerating the samples again and see whether the backslashes still appear. I can also try generating the samples using a Linux environment, which will probably alleviate the issue. Please let me know what needs to happen in order for this pull request to move forward. Thanks!

Reply:
I went ahead and regenerated the samples on Linux. This seems to have fixed the file separator issue, so I hope that we can move forward with the changes :)

Reply:
👍 

will try to get it merged in the next few days.